.github/copilot-instructions.md

```diff
@@ -0,0 +1,114 @@
+- [ ] **BEM Naming**: Ensure BEM (Block Element Modifier) conventions are followed
+- [ ] **Semantic HTML**: Verify use of semantic HTML5 elements
+- [ ] **Accessibility**: Check WCAG 2.2 AA compliance including:
+  - ARIA roles and attributes where necessary
+  - Proper contrast ratios
+  - Keyboard navigation support
+  - Focus states for interactive elements
+
+### For CSS/SCSS Changes
+- [ ] **Dist Regeneration**: Verify all CSS files under dist folder have been regenerated
+- [ ] **Browser Testing**: Check that UI has been tested in all supported browsers
+- [ ] **Visual Regression**: Ensure Percy build was done and approved for visual regression testing
+- [ ] **Dark Mode**: Verify UI works in dark mode
+- [ ] **RTL Support**: Check right-to-left (RTL) mode compatibility
+- [ ] **Storybook**: Refer to the "Code Quality Checks" section for detailed Storybook requirements.
+- [ ] **BEM Naming**: Ensure BEM methodology is followed for all selectors
+- [ ] **CSS Variables**: Check use of CSS variables for colors, spacing, and typography
+- [ ] **Responsive Design**: Verify mobile-friendly design using CSS Grid or Flexbox
+- [ ] **Responsive Breakpoints**: Verify the proper use of responsive breakpoints at the proper sizes (320px, 512px, 768px, 1024px, 1280px, 1440px, 1680px, 1920px).
+
+## Code Quality Checks
+
+### Storybook Requirements
+- **Required Stories**: All components must include RTL and Text Spacing stories unless explicitly excluded
+- **Story Coverage**: Verify that new/modified components have appropriate Storybook stories
+- **Story Quality**: Check that stories demonstrate key component variants and states
+- **Documentation**: Ensure stories include proper controls and documentation
+
+### General Code Quality
+- **Consistency**: Check adherence to existing code patterns and style
+- **Documentation**: Verify appropriate comments and documentation
+- **Performance**: Look for potential performance issues
+- **Security**: Check for any security vulnerabilities
+
+### Component-Specific
+- **API Consistency**: Ensure component APIs follow established patterns
+- **Props/Attributes**: Verify proper typing and validation
+- **Event Handling**: Check appropriate event handling patterns
+- **State Management**: Review state management approaches
+
+### Testing
+- **Test Coverage**: Ensure adequate test coverage for new/modified code
+- **Test Quality**: Review test cases for completeness and correctness
+- **Integration Tests**: Check that integration tests cover user scenarios
+
+## Style Guide Compliance
+Reference the official style guide: https://github.com/eBay/evo-web/blob/main/packages/skin/STYLEGUIDE.md
+
+## Accessibility Priority
+Always prioritize accessibility (a11y) compliance:
+- WCAG 2.2 AA guidelines adherence
+- Keyboard navigation support
+- Screen reader compatibility
+- Proper color contrast
+- Focus management
+
+## Package-Specific Considerations
+
+### Skin Package
```

Comment:
Could we also ask it to verify that Skin `feat` PRs include updated DS metadata in the associated module's `+page.marko` file?